templates/webapi/test/infopanel.html.ep

```diff
@@ -46,12 +46,14 @@
                     % else {
                         % my $state_detail = '';
                         % if (my @gru_deps = $job->gru_dependencies) {
+                            % my $minion_id = $gru_deps[0] ? $gru_deps[0]->gru_task_id : "";
```

Comment:
There could be more than one gru_task theoretically. Why do you only add a link for the first?